std/complex.d

```diff
@@ -292,14 +292,14 @@ struct Complex(T)  if (isFloatingPoint!T)
             // r = lhs
             // theta = 0
             ab = lhs ^^ this.re;
-            ar = log(lhs) * this.im;
+            ar = std.math.log(lhs) * this.im;
         }
         else
         {
             // r = -lhs
             // theta = PI
-            ab = (-lhs) ^^ this.re * exp(-PI * this.im);
-            ar = PI * this.re + log(-lhs) * this.im;
+            ab = (-lhs) ^^ this.re * std.math.exp(-std.math.PI * this.im);
+            ar = std.math.PI * this.re + std.math.log(-lhs) * this.im;
```

Comment:
Yeah `import std.math : cos, exp, log, sin` would be a bit nicer, in fact since hijacking of local symbols was fixed `import std.math` would be an option as well.